x/feemarket/keeper/keeper.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2021 Evmos Foundation
+// This file is part of Evmos' Ethermint library.
+//
+// Evmos is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Evmos packages are distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Ethermint library. If not, see https://github.com/evmos/evmos/blob/main/LICENSE
+package keeper
+
+import (
+	"math/big"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/evmos/ethermint/x/feemarket/types"
+)
+
+// KeyPrefixBaseFeeV1 TODO: Temporary will be removed with params refactor PR
+var KeyPrefixBaseFeeV1 = []byte{2}
```

Comment:
@fedekunze how will this be addressed - https://github.com/evmos/ethermint/issues/1562

Reply:
needs a larger effort to support clients that are able to route queries to legacy versions